doc/56_Search.md

```diff
@@ -13,13 +13,13 @@ There are several ways to perform a search and to find things in the workspace:
 ## Search in Workspace
 
 The `Search in Workspace` command will bring up the `Search` view. Run it either through the
-command palette or by pressing <kbd>Ctrl+Shift+F</kbd> (<kbd>⌘+Shift+F</kbd> on macOS)
+command palette or by pressing <kbd>Ctrl+Shift+F</kbd> (<kbd>⌘+Shift+F</kbd> on macOS).
 
 In the `Search` view, you can start typing, and the matching results will be shown automatically
 without pressing <kbd>Enter</kbd>. The search can be restricted to case matching, whole word
-matching, or, when desired, the query can be given as a regular expression. Search results are
+matching, or the query can be given as a regular expression. Search results are
 grouped by file, with an indication of the number of occurrences and the file's path. Expand a file
-to see a preview of all of the hits within that file. Then single-click on one of the hits to view
+to see a preview of all of the hits within that file. Then single-click on one of the files to view
```

Comment:
'hits' are subnodes of the 'files', so better leave it as it was

Reply:
OK, fixed in the follow-up